srcpkgs/python3-ijson/patches/remove-yajl-backend.patch

```diff
@@ -0,0 +1,62 @@
+--- ijson/backends/yajl.py.orig	2021-03-02 05:04:56.000000000 -0500
```

Comment:
Could you instead simply remove the file in `post_patch`?

`rm ijson/backends/yajl.py`

Kinda weird that we build the yajl stuff only to completely remove it :[

Reply:
`yajl.py` is the backend for yajl 1.x. There are 3 other backends left intact to interact with the version of yajl already packaged (2.x). I was removing this because simply having the patch to remove the backend from `test/test_base.py` wasn't enough to pass the check phase.